lib/docs/filters/gnu_cobol/clean_html.rb

```diff
@@ -0,0 +1,72 @@
+module Docs
+  class GnuCobol
+    class CleanHtmlFilter < Filter
+      def call
+        # Replace the title
+        at_css('.settitle').content = 'GnuCOBOL'
+
+        # Remove the Table of Contents
+        # It's huge and the DevDocs sidebar is basically a direct copy
+        css('.contents, .contents-heading').remove
+
+        # Remove the changelog
+        at_css('p').remove
+        at_css('ol').remove
+
+        # Remove horizontal lines
+        css('hr').remove
+
+        # Remove everything after Appendix B
+        # This includes the license text, the document changelog, the compiler changelog and the footnote
+        start_element = at_css('a[name="Appendix-C-_002d-GNU-Free-Documentation-License"]').previous_element
+        next_element = start_element.next_element
+        until start_element.nil?
+          start_element.remove
+          start_element = next_element
+          next_element = start_element.nil? ? nil : start_element.next_element
+        end
```

Comment:
This seems to remove the content of Appendix B, too. However, appendix B is just a list of all the reserved words, so it might be reasonable to remove it too.